### PR TITLE
Add: modal component

### PR DESCRIPTION
**Pull request by devsurajrai**
Added the modal component the library. Please review.

Screenshot 2022-04-03 at 7 37 39 PM

### CODE REVIEW

**Review by Dev-AkshatGupta — APPROVED** [review]
This looks quite good to me nothing to suggest in this

File: components/docs/docs.html [hunk 1]
```diff
@@ -65,6 +65,9 @@ <h1 class="component-heading">Components</h1>
           <li class="menu-item">
             <a name="chips" class="component-btn  a-no-underline" href="#">Navigation</a>
```
Comment (SJTGSHIVAM) [inline components/docs/docs.html:66]: when attributes are long can have them on multiple line for better readabilty.

**Review by SJTGSHIVAM — APPROVED** [review]
No major issues were found. Looks clean. I believe its good to merge

File: components/modal/modal.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const unHideModalButton = document.querySelector(".modal-button-unhide");
+const hideModalButton = document.querySelector(".modal-button-hide");
+const modal = document.querySelector(".modal");
+
+const handleModalUnhide = () => {
+  modal.classList.remove("modal-dispay-none");
```
Comment (saifudinbadani) [inline components/modal/modal.js:6]: this code is easy to read and understand.

**Review by saifudinbadani — APPROVED** [review]
Found no issues in the code.

File: components/docs/docs.html (hunk 1 above)
Comment (devsurajrai) [reply components/docs/docs.html:66]: understood

**Pull request merged**